tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "outDir": "./dist/",        // path to output directory
     "sourceMap": true,          // allow sourcemap support
     "strict": true,             // enable strict type checks as a best practice
-    "module": "es6",            // specify module code generation
+    "module": "esnext",         // specify module code generation
```

Comment:
Why did we have to switch to esnext? Do you think it will have impact in rest of the codebase?

Reply:
I saw an explanation for this in paJDYF-eod-p2 👍 Nice discussion.

Reply:
If we are to go this route, the suggestion in the above post is to use es2020.

Reply:
Thank you Anurag. We are still discussing the best approach to handle it.